class.frame-nonce-preview.php

```diff
@@ -95,6 +101,19 @@ public function set_post_to_publish( $posts ) {
 
 		return $posts;
 	}
+
+	/**
+	 * Handle validation for autosave preview request
+         *
```

Comment:
Changes look good and tests well.  Could you please just fix the wonky indentation in this block?  Thanks!